code/modules/clothing/head/helmet.dm

```diff
@@ -527,6 +527,11 @@
 
 	name = "[K.name_set] plated helmet"
 	slowdown = K.slowdown_set
-	armor = K.armor_set
+	if (islist(armor) || isnull(armor))		//For an explanation see code/modules/clothing/under/accessories.dm#L39 - accessory detach proc							
+		armor = getArmor(arglist(armor))
+	if (islist(armor) || isnull(armor))
+		armor = getArmor(arglist(armor))
+
+	armor = armor.attachArmor(K.armor)
```

Comment:
from what i read this adds armor, not sets it, which introduces a minor unintentional buff

Reply:
https://github.com/yogstation13/Yogstation/blob/master/code/datums/armor.dm#L62

Reply:
Yeah I reduced the melee armor on all of them by 5 and as of the last commit the fire by 10 because I forgot that part